src/main/java/persistence/sql/ddl/TableQueryMapper.java

```diff
@@ -0,0 +1,88 @@
+package persistence.sql.ddl;
+
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import jakarta.persistence.Transient;
+
+import java.lang.reflect.Field;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class TableQueryMapper {
+    private static final String SPACE = " ";
+    private static final String CREATE_QUERY_FORMAT = "create table %s";
+    private static final String DROP_QUERY_FORMAT = "drop table %s";
+    private static final String ID_QUERY_FORMAT = "%s primary key";
+    private static final String COLUMN_QUERY_FORMAT = "(%s, %s)";
+    private static final String COLUMN_SEPARATOR = ", ";
+    private final ColumnMapper columnMapper;
```

Comment:
```suggestion
    private static final String COLUMN_SEPARATOR = ", ";
    
    private final ColumnMapper columnMapper;
```

상수와 상태값을 구분지어주면 코드를 보는데 조금 더 편할 것 같은 느낌이 드네요!